tests/e2e-inst-test.sh

```diff
@@ -254,7 +280,7 @@ for TEST in $TESTS; do
     fi
 
     info
-    if [[ $found -eq 1 && $errors -eq 0 ]]; then
+    if [[ $found -eq 1 ]]; then
         info "$TEST: SUCCESS"
```

Comment:
It would be easy to manage the output of results to be continuous?

Current:

https://github.com/aquasecurity/tracee/actions/runs/15976407193/job/45059896815?pr=4810#step:4:1127

```
INFO: = CHECKING TESTS RESULTS ======================================
INFO: 
INFO: 
INFO: PROCESS_EXECUTE_FAILED: SUCCESS
INFO: 
INFO: 
INFO: VFS_WRITE: SUCCESS
INFO: 
INFO: 
INFO: FILE_MODIFICATION: SUCCESS
...
```

Would be better as:

```
INFO: = CHECKING TESTS RESULTS ======================================
INFO: 
INFO: PROCESS_EXECUTE_FAILED: SUCCESS
INFO: VFS_WRITE: SUCCESS
INFO: FILE_MODIFICATION: SUCCESS
...
```